Lib/test/datetimetester.py

```diff
@@ -2425,6 +2425,22 @@ def test_strptime(self):
                 self.assertEqual(expected, got)
 
         strptime = self.theclass.strptime
+
+        # bpo-34903: Check that single digit dates and times are allowed.
+        with self.assertRaises(ValueError):
+            # %y does require two digits
+            newdate = strptime('1/2/3 4:5:6', '%d/%m/%y %H:%M:%S')
+        inputs = [
```

Comment:
Could you provide comments specifying which format option you're specifically testing? Or better yet, provide explicit tests for each format you're testing? (You can keep these real-world examples as well).